source/transactions/tests/retryable-commit.yml

```diff
@@ -252,16 +252,16 @@ tests:
       collection:
         data: []
 
-  - description: commitTransaction fails after WriteConcernError Interrupted
+  - description: commitTransaction is not retried after UnsatisfiableWriteConcern error
 
     failPoint:
       configureFailPoint: failCommand
       mode: { times: 1 }
       data:
           failCommands: ["commitTransaction"]
           writeConcernError:
-            code: 11601
-            errmsg: operation was interrupted
+            code: 100
```

Comment:
I think 11601 should still produce a UnknownTransactionCommitResult - so opted to use 100 which explictly is not retryable and won't result in a UnknownTransactionCommitResult.